### PR TITLE
Главная страница проектов

### PR DESCRIPTION
**Pull request by NikitaGoroshko7**
https://trello.com/c/4fcUETwP/15-%D0%B3%D0%BB%D0%B0%D0%B2%D0%BD%D0%B0%D1%8F-%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0-%D0%BF%D1%80%D0%BE%D0%B5%D0%BA%D1%82%D0%BE%D0%B2

Has been replaced class on id due to incorrect display in style ForgotAndResetPage and in other Views for change password
Has been added Layout for web application
Has been added main page for projects
Has been changed endpoint in a class startup

### CODE REVIEW

File: CrowdFundingApplication/Startup.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllerRoute(
                     name: "default",
-                    pattern: "{controller=Home}/{action=Index}/{id?}");
+                    pattern: "{controller=Home}/{action=MainPage}/{id?}");
```
Comment (GalayM) [inline CrowdFundingApplication/Startup.cs]: Why this change?

File: CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css [hunk 2]
```diff
@@ -4,16 +4,9 @@ body {
...
-    height: 100vh;
-    display: flex;
-    align-items: center;
-    justify-content: center;
-    background-size: cover;
-    background-position: center;
 }
 
-.main {
+#main {
```
Comment (GalayM) [inline CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css]: Another ID? Why?

File: CrowdFundingApplication/Views/Home/MainPage.cshtml [hunk 3]
```diff
@@ -1,5 +1,104 @@
...
+                        <a class="small" href="#">Категория</a>
+                        <h6 class="card-title mt-1">Название</h6>
+                        <p class="card-text">Краткое описание проекта для которого необходимо собрадь деньги. Просто обычное описание какого-то проекта</p>
+                    </div>
+
+                    <div class="card-footer">
+                        <div class="progress" style="height: 3px;">
+                            <div class="progress-bar" role="progressbar" style="width: 25%;" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100"></div>
+                        </div>
+                        <br />
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Home/MainPage.cshtml]: BR is added only in solid text to break lines

File: CrowdFundingApplication/Views/Home/MainPage.cshtml [hunk 4]
```diff
@@ -1,5 +1,104 @@
...
+                        <span class="" style="font-weight: bold; color: #007bff">25% </span> <br />
+                        <span style="color: gray">Собрано</span>
+                    </div>
+
+                </div>
+            </li>
+
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Home/MainPage.cshtml]: ???

File: CrowdFundingApplication/Views/Shared/_Layout.cshtml [hunk 5]
```diff
@@ -3,43 +3,69 @@
...
-            &copy; 2022 - CrowdFundingApplication - <a asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+            <div class="row">
+                <div class="col-12">
+                    <div class="mb-0">
+                        <a href="#">Github</a>
+                        <a href="#">Linkedin</a>
+                        <a href="#">Facebook</a>
+                        <a href="#">Instagram</a>
+                    </div>
+                    <p class="mb-0 py-3">&copy; 2022 - Design by Nikita. All rights reserved. </p>
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Shared/_Layout.cshtml]: Why is there a space after the dot?

File: CrowdFundingApplication/Views/Shared/_Layout.cshtml [hunk 6]
```diff
@@ -3,43 +3,69 @@
...
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
-                        </li>
-                        <li class="nav-item">
-                            <a class="nav-link text-dark" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
-                        </li>
-                    </ul>
-                </div>
+<body class="bg-light">
+
+    @*NAVBAR*@
```
Comment (GalayM) [inline CrowdFundingApplication/Views/Shared/_Layout.cshtml]: ???

File: CrowdFundingApplication/Startup.cs (hunk 1 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Startup.cs]: When a user for the first time on this web-app or unauthorized we allow use only welcome page where the user can register or authorize. When the user is already registered in web-app this is unnecessary to login again and we allow the user start with MainPage

File: CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css (hunk 2 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css]: No, this is the previous id. I've replaced class on id due to incorrect display in style ForgotAndResetPage and in other Views for change password

File: CrowdFundingApplication/Views/Home/MainPage.cshtml (hunk 3 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Home/MainPage.cshtml]: Has been deleted

File: CrowdFundingApplication/Views/Home/MainPage.cshtml (hunk 4 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Home/MainPage.cshtml]: Has been deleted the empty space

File: CrowdFundingApplication/Views/Shared/_Layout.cshtml (hunk 5 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Shared/_Layout.cshtml]: Has been deleted the empty space after the dot

File: CrowdFundingApplication/Views/Shared/_Layout.cshtml (hunk 6 above)
Comment (NikitaGoroshko7) [reply CrowdFundingApplication/Views/Shared/_Layout.cshtml]: Has been deleted

File: CrowdFundingApplication/Startup.cs (hunk 1 above)
Comment (GalayM) [reply CrowdFundingApplication/Startup.cs]: There should be a reverse logic, the main page should become an Index

File: CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css (hunk 2 above)
Comment (GalayM) [reply CrowdFundingApplication/wwwroot/css/ForgotAndResetPage.css]: This is not the right solution, use classes

**Review by GalayM — APPROVED** [review]
